core/src/main/java/org/apache/accumulo/core/Accumulo.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core;
+
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.client.impl.AccumuloClientImpl;
+
+/**
+ * The main entry point for Accumulo public API.
+ */
+public final class Accumulo {
+
+  private Accumulo() {}
```

Comment:
Why create this class, instead of having a static method on AccumuloClient?

Reply:
The idea was to have one entry point for interacting with Accumulo API.  We could put other static methods here for creating RFiles, Iterators, etc.  Since we don't have a nice package just for our public API this would help with an easy way to interact with Accumulo, beyond just creating a client.

Reply:
Ok that seems like a nice goal.  I think it would be nice to document this a bit better in the javadoc.  The javadoc states that this is the main entry point for the API, but that is not a true statement.  Currently there are other entry points outside of this class.  I think it would be useful to users and Accumulo developers to communicate this nuance of the goal vs reality.  

Even though not all API entry points are here in code, we could add javadocs links for those here.  Then from this one class either via code or JAvadoc all top level public APIs can be found.  We could also put the info from the README about what constitutes public API in this calsses javadoc.  

I think these javadoc updates can be done as follow on work.  I can make these changes or review them if someone else is interested in making the changes.